ui/src/environments/environment-type.ts

```diff
@@ -53,4 +57,6 @@ export interface Environment {
   // See RW-1966 for details.
   // Exit criteria: remove flag for Athens release.
   enableJupyterLab: boolean;
+
+  enableComplianceLockout: boolean;
```

Comment:
I'm not sure whether `compliance` captures it. Maybe more broadly: `enableAthensRegistration`